packages/contrast-colors/index.js

```diff
@@ -36,6 +36,20 @@ function cArray(c) {
   return new Array(L, U, V);
 }
 
+function removeDuplicates(originalArray, prop) {
+  var newArray = [];
+  var lookupObject  = {};
+
+  for(var i in originalArray) {
```

Comment:
You should use `for(i of lookupObject)` to iterate over arrays, `for...in` is for Objects. See [for..of docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of)

Reply:
Changing this to a for...of breaks